src/rules/root-no-standard-properties/index.js

```diff
@@ -4,6 +4,8 @@ import {
   validateOptions,
 } from "../../utils"
 
+const RE_VALID_CSS_PROPERTY = /^[a-z-]+$/
```

Comment:
I’ll wait until you decide whether to check for `@`/`$` or the RegExp and than either fix this or rollback the RegExp.